main.go

```diff
@@ -19,6 +19,7 @@ var (
 	portFlag       = kingpin.Flag("port", "Port to listen on.").Short('p').Default("9130").String()
 	fsMetricFlag   = kingpin.Flag("filesystem-metrics", "Export filesystem and usage data metrics for each user and group.").Default("false").Bool()
 	insecureFlag   = kingpin.Flag("insecure", "Disable the verification of the SSL certificate").Default("false").Bool()
+	apiVersionFlag = kingpin.Flag("apiVersion", "API version to query the flashblade").Default("1.7").String()
```

Comment:
`kingpin.Flag("api-version", "API version to query the flashblade")` would be more consistent with the casing and style of the other flags. Thanks for providing a good default.